### ISSUE TITLE
Incorrect positioning of bullet tooltip when using JSON config

### ISSUE THREAD

**Issue by martynasma** [amcharts/amcharts4#86]
This happens only with JSON config. Possibly because JSON processor is using `bullets.create(CircleBullet)` instead of `bullets.push(new CircleBullet())`.

![image](https://user-images.githubusercontent.com/1458520/41548881-d5e51e9c-732c-11e8-9140-face23958060.png)

Example: https://codepen.io/team/amcharts/pen/80cab78395c5026f40a03085d366410e?editors=0010

**Comment by Ekliptor** [comment]
I am having this bug without using JSON config. Specifically without using am4core.createFromConfig() on an XYChart with a line series.
I get set the data directly in TypeScript with:
```
chart.data = [];
for (let i = 0; i < dataPoints.length; i++)
{
       let series = {xVal: dataPoints[i]}
       // modify my data
       chart.data.push(series);
}
```
See Request #37513 for my full sample code

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.31](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.31).

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
